test/NuGet.Core.Tests/NuGet.Packaging.Test/ContentModelTests/PatternExpressionMatchTests.cs

```diff
@@ -0,0 +1,198 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using NuGet.ContentModel;
+using NuGet.ContentModel.Infrastructure;
+using Xunit;
+
+namespace NuGet.Client.Test
+{
+    public class PatternExpressionMatchTests
+    {
+
+        
+        [Fact]
+        public void LiteralSegment_OnlyShouldMatch()
+        {
+            // match literal segments given a pattern
+            Dictionary<string, ContentPropertyDefinition> propertyDefinitions =
+                new Dictionary<string, ContentPropertyDefinition>() {
+                {"type", new ContentPropertyDefinition("text") }
+            };
```

Comment:
https://github.com/NuGet/NuGet.Client/blob/dev/docs/coding-guidelines.md#c-coding-style

1.  We use Allman style braces.
2.  There should be 1 space between a curly brace and whatever follows on the same line.
3.  [Optional] Since this is a `new` expression the data type of the right-hand side (RHS) is obvious.  You could have the explicit type just once (LHS or RHS).

```suggestion
            Dictionary<string, ContentPropertyDefinition> propertyDefinitions = new()
            {
                { "type", new ContentPropertyDefinition("text") }
            };
```

This feedback applies throughout the PR.